qa-content/qa-admin.js

```diff
@@ -112,6 +112,27 @@ function qa_mailing_start(noteid, pauseid)
 	);
 }
 
+function qa_update_dom(response)
+{
+	if (!response.hasOwnProperty('domUpdates')) {
+		return;
+	}
+
+	for (var i = 0; i < response.domUpdates.length; i++) {
```

Comment:
Please use `let`

Reply:
I have already answer you about `let` in another comment which you didn't reply

Reply:
Sorry, I don't saw it :) But, you can think to rewrite code to ES6;)